vnext/Desktop.UnitTests/UnicodeConversionTest.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include <CppUnitTest.h>
 #include <string>
-#include "../ReactWindowsCore/unicode.h"
+#include "../Common/unicode.h"
```

Comment:
If this is already part of the include path, maybe use `#include <unicode.h>`?

Reply:
Fixed.

---
In reply to: [333300924](https://github.com/microsoft/react-native-windows/pull/3382#discussion_r333300924) [](ancestors = 333300924)